proto/public/uber/cadence/api/v1/history.proto

```diff
@@ -90,31 +90,29 @@ message HistoryEvent {
 
 message WorkflowExecutionStartedEventAttributes {
```

Comment:
the fairly-large similarity between this and `WorkflowExecutionInfo` had me comparing things out of curiosity for a while... they may not _all_ make sense to exist in `WorkflowExecutionInfo` (e.g. identity is per-event, not per-workflow), but I wonder if some of the missing fields would be useful to bring over.

e.g. when listing or describing a workflow:
- input/result seem potentially useful (though possibly large data on list endpoints - defaulting to null makes sense, could be filled in when explicitly requested)
- original / continue run_id seems like something that should exist
- cron schedule and retry policy are almost definitely relevant to people looking at a Describe
- *_start_to_close_timeouts are workflow-wide, possibly relevant.  (... though now that I think about it, it's a little strange that those are not modify-able as a workflow runs.  the use would be fairly rare, probably?)